modules/serverless-ossm-enabling-serving-metrics.adoc

```diff
@@ -37,22 +37,19 @@ spec:
 +
 This step prevents metrics from being disabled by default.
 
-. Apply the following network policy to allow traffic from the Prometheus namespace:
++ 
+The same mechanism is used for Knative Eventing. To enable metrics for Knative Eventing, you need to specify `prometheus` as the `metrics.backend-destination` in the `observability` spec of the Knative Eventing custom resource (CR) as follows: 
 +
 [source,yaml]
 ----
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
 metadata:
-  name: allow-from-openshift-monitoring-ns
-  namespace: knative-serving
+  name: knative-eventing
 spec:
-  ingress:
-  - from:
-    - namespaceSelector:
-        matchLabels:
-          name: "openshift-monitoring"
-  podSelector: {}
+  config:
+    observability:
+      metrics.backend-destination: "prometheus"
```

Comment:
Why are we removing the network policy?

Shouldn't the documentation of the new feature be an additive section ?

Reply:
I see why we removed it

> Right now the second bullet is not needed because we automatically generate the "allow-from-openshift-monitoring-ns" network policy at the Serverless operator side. So the section starting with the following should be removed:
"Apply the following network policy to allow traffic from the Prometheus namespace:..."

However, we're not really documenting how to disable the network policy generation with the annotation

Reply:
To document that part I think in a new section "Disable the default networkpolicies" we can list that by annotating KnativeEventing and KnativeService CR with `serverless.openshift.io/disable-istio-net-policies-generation` the default network policies are not applied by the operator

Reply:
@pierDipi, Can you please provide the sample code block with `erverless.openshift.io/disable-istio-net-policies-generation` so that I can include in the section?

Reply:
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeEventing # Same for kind: KnativeServing
metadata:
  name: knative-eventing
  namespace: knative-eventing
  annotations:
    serverless.openshift.io/disable-istio-net-policies-generation: "true"
```

Reply:
As per your suggestion, I have added a new section for "Disabling the network policies".